src/index.html

```diff
@@ -22,6 +22,342 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo_logo.img"
```

Comment:
For logos, use the format: "[Organization name] logo." 

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/757148b5-cc33-4715-987d-e7f066fa3a68)


```suggestion
          alt="Moyo logo"
```